addons/project/controllers/portal.py

```diff
@@ -407,9 +407,9 @@ def get_grouped_tasks(pager_offset):
                 if group == 'milestone_id':
                     grouped_tasks = [Task_sudo.concat(*g) for k, g in groupbyelem(tasks_project_allow_milestone, itemgetter(group))]
 
-                    if not grouped_tasks:
+                    if not grouped_tasks and tasks_no_milestone:
                         grouped_tasks = [tasks_no_milestone]
```

Comment:
```suggestion
                    if not grouped_tasks
                        if tasks_no_milestone:
                            grouped_tasks = [tasks_no_milestone]
```
so that no elif needed, and we don't check the same thing twice :slightly_smiling_face: